tests/common/test_logger.py

```diff
@@ -466,7 +467,8 @@ def setUp(self):
         self.event_dir = os.path.join(self.lib_dir, EVENTS_DIRECTORY)
         fileutil.mkdir(self.event_dir)
 
-        self.log_file = tempfile.mkstemp(prefix="logfile-")[1]
+        self.log_folder = tempfile.mkdtemp()
+        self.log_file = tempfile.mkstemp(prefix="logfile-", dir=self.log_folder)[1]
```

Comment:
From [the docs](https://docs.python.org/3/library/tempfile.html): "mkstemp() and mkdtemp() are lower-level functions which require manual cleanup."

Reply:
Good to know. Added the cleanup in this test.